src/compose/ComposeBox.js

```diff
@@ -108,6 +110,21 @@ export default class ComposeBox extends PureComponent {
     this.clearMessageInput();
   };
 
+  handleEdit = () => {
+    const { auth, editMessage, actions } = this.props;
+    const { message } = this.state;
+    if (editMessage.content !== message) {
+      try {
+        patchMessage(auth, replaceEmoticonsWithEmoji(message), editMessage.id, (msg: string) =>
```

Comment:
It was not working, so I thought of adding callback function.
Now refactored it a bit and is now working 👍 .